lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwift.h

```diff
@@ -116,6 +116,11 @@ class TypeSystemSwift : public TypeSystem {
 
   static LanguageSet GetSupportedLanguagesForTypes();
   virtual SwiftASTContext *GetSwiftASTContext() const = 0;
+
+  std::function<SwiftASTContext *()> GetSwiftASTContextProvider() const {
```

Comment:
Is either of `GetSwiftASTContextLazy()` or `GetLazySwiftASTContext()` a more descriptive name?